src/Sylius/Behat/Context/Setup/SecurityContext.php

```diff
@@ -28,32 +31,60 @@
     private $securityService;
 
     /**
+     * @var Session
+     */
+    private $minkSession;
+
+    /**
+     * @var array
+     */
+    private $minkParameters;
+
+    /**
      * @var HomePage
      */
     private $homePage;
 
     /**
-     * @var Session
+     * @var TestUserFactoryInterface
      */
-    private $minkSession;
+    private $testUserFactory;
 
     /**
-     * @var array
+     * @var RepositoryInterface
```

Comment:
IMHO typehinting to `UserRepositoryInterface` would be wise, because user repository should always be implementing that interface and if we will need to use any of the user repository specific method in the future we won't have to do any irrelevant changes.